commands/anonSuggest.js

```diff
@@ -0,0 +1,19 @@
+require('discord.js');
+
+module.exports = {
+	name: 'anonsuggest',
+	description: 'Posts an anonymous Suggestion',
+	usage: 'anonsuggest',
+	helpMsg: true,
+	execute(message, args) {
+		const {modBot} = message.client.config.channelIDs;
+
+		console.log('running');
```

Comment:
No

Reply:
Oops, I removed the rest. Missed that one clearly.